godtools/Views/TractElements/Header.swift

```diff
@@ -26,6 +26,8 @@ class Header: BaseTractElement {
         }
     }
     
+    var includesNumber = false
```

Comment:
Yes, actually I'm using it on the classes Number and Title (which are children of Header), but, thinking on a better way to handle it... There should be a better way, will think about it (will make an issue), but I believe I should do that in other branch